### ISSUE TITLE
Change user avatar 

### ISSUE THREAD

**Issue by yfeizi** [guardianproject/ChatSecureAndroid#647]
Is this available to user change its avatar image and also display itself avatar in chat ?

**Comment by benefaktorix** [comment]
Yes and no at the same time. You can't do it via app itself, but as a workaround you can download Pidgin(windows) or Adium(mac) and put the avatar on your account there which then will also appear on you chatasecure account and in chats

**Comment by yfeizi** [comment]
No way to do it with chatasecure? How Pidgin or ... implement it ?

**Comment by Joseph2106** [comment]
Any way u want I'm here to satisfy

**Comment by Cedybt** [comment]
Its so damn Possible via CS!!!
